src/mlpack/methods/ann/augmented/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Define the files we need to compile
+# Anything not in this list will not be compiled into mlpack.
+set(SOURCES
+)
+
+# Add directory name to sources.
+set(DIR_SRCS)
+foreach(file ${SOURCES})
+  set(DIR_SRCS ${DIR_SRCS} ${CMAKE_CURRENT_SOURCE_DIR}/${file})
+endforeach()
```

Comment:
If there are no sources in this directory, I think you can remove all the lines above this comment and change below to

```
set(MLPACK_SRCS ${MLPACK_SRCS} PARENT_SCOPE)
```

but I think also that line should come after the recursion into the `tasks/` subdirectory.